src/constants.tsx

```diff
@@ -142,7 +142,7 @@ export const OPENSRP_TEMPLATE_ENDPOINT = 'template';
 export const OPENSRP_PLANS_BY_USER_FILTER = 'plans/user';
 export const OPENSRP_V1_SETTINGS_ENDPOINT = 'settings/sync';
 export const OPENSRP_V2_SETTINGS = 'v2/settings/';
-export const OPENSRP_JURISDICTION_HIERARCHY_ENDPOINT = 'location/getHierarchy';
+export const OPENSRP_JURISDICTION_HIERARCHY_ENDPOINT = 'location/hierarchy';
 export const OPENSRP_PLAN_HIERARCHY_ENDPOINT = 'location/getHierarchy/plan';
```

Comment:
lets also change this hierarchy endpoint  `location/getHierarchy/plan` => `location/hierarchy/plan`